examples/cigarette_smokers.rb

```diff
@@ -218,7 +218,7 @@ def refill_table
 
   def note_smokers
     active_smokers = @table.smokers.select(&:smoking?)
-    puts "*** Active smokers: #{active_smokers.map(&:name).join(", ")}"
+    puts "*** Active smokers: #{active_smokers.map(&:name).join(', ')}"
```

Comment:
It's a standard RuboCop offense - mostly related to Ruby's default style guide (using single quotes instead of double quotes for constant strings).